acceptance/repository-cloudant/src/__tests__/cloudant-default-repository.acceptance.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/test-repository-cloudant
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {DefaultTransactionalRepository} from '@loopback/repository';
+import {
+  CrudRepositoryCtor,
+  crudRepositoryTestSuite,
+} from '@loopback/repository-tests';
+import {CLOUDANT_CONFIG, CLOUDANT_FEATURES} from './cloudant.datasource';
+
+describe('CLOUDANT + DefaultCrudRepository', () => {
+  crudRepositoryTestSuite(
+    CLOUDANT_CONFIG,
+    // Workaround for https://github.com/microsoft/TypeScript/issues/31840
+    DefaultTransactionalRepository as CrudRepositoryCtor,
```

Comment:
Since it [doesn't support transactions](https://github.com/strongloop/loopback-next/pull/3968/files#diff-b64c9eeda3125036291ac7b928f708a6R22), I think you should change this to `DefaultCrudRepository` like in the MongoDB one.

Reply:
👍 Cloudant uses `_rev` to handle the update conflict, it doesn't handle transaction like sql dbs.

Reply:
right, I just copied it from MySQL :p